locale/templates/settings.ftl

```diff
@@ -539,6 +539,7 @@ tfa-row-change-modal-explain = You won’t be able to undo this action.
 auth-error-102 = Unknown account
 auth-error-103 = Incorrect password
 auth-error-110 = Invalid token
+auth-error-114 = You've tried too many times. Please try again { $retryAfter }.
```

Comment:
Need to add a variable at the beginning of this block what { $retryAfter } is for